install/share/opendnssec_conf.template

```diff
@@ -8,7 +8,6 @@
 			<Module>$SOFTHSM_LIB</Module>
 			<TokenLabel>$TOKEN_LABEL</TokenLabel>
 			<PIN>$PIN</PIN>
-            <AllowExtraction/>
```

Comment:
Why is this removed? Does it work in multireplica scenario without extractable tokens?